ember-flight-icons/app/helpers/substring.js

```diff
@@ -0,0 +1 @@
+export { default } from '@hashicorp/ember-flight-icons/helpers/substring';
```

Comment:
question: do we want to expose this helper to consumers of the addon?

Reply:
no, they don't need it. we won't (eventually) need it either.

Reply:
not a blocker because I may be mis-remembering, but isn't there a way to create a helper just for the dummy app?